src/Sulu/Bundle/AdminBundle/Resources/js/containers/Link/README.md

```diff
@@ -0,0 +1,29 @@
+The `Link` component is a universal link selection and can additionally also be used for external links.
```

Comment:
```suggestion
The `Link` component implements a universal link selection that allows to select internal and external links.
It can be extended with additional custom link types via the `linkTypeRegistry`.

```